traitlets/traitlets.py

```diff
@@ -372,6 +372,8 @@ def __init__(self, default_value=Undefined, allow_none=None, read_only=None, hel
         if help is not None:
             self.metadata['help'] = help
 
+        self.default_handlers = []
```

Comment:
Since there are already `_trait_notifiers`, `_trait_validators` and `_trait_values`, this could be made private and called `_trait_default_generators` or something like this.